components/breadcrumb/stories/template.js

```diff
@@ -33,12 +36,14 @@ export const Template = ({
 				})}
 			>
 				${items.map((item, idx, arr) => {
-					const { label, isDisabled, isDragged, iconName } = item;
+					const { label, isDisabled, iconName } = item;
 					return html` <li
 						class=${classMap({
 							[`${rootClass}-item`]: true,
 							"is-disabled": isDisabled,
-							"is-dragged": isDragged,
+							"is-dragged": isDragged && item.isDragged,
+							"is-focused": isFocused && item.isFocused,
```

Comment:
My hope in including `is-focused` was that Chromatic would capture that there isn't a focus style to give us coverage of that, but I did worry that it might be confusing for the user to toggle `isFocused` to `true` and see no visual change. I'm good to remove these.